migrate/repo_checks.py

```diff
@@ -0,0 +1,364 @@
+"""
+Run checks Against Repos and correct them if they're missing something.
+
+"""
+from itertools import chain
+from pprint import pformat
+
+import click
+import requests
+from fastcore.net import HTTP4xxClientError, HTTP5xxServerError, HTTP404NotFoundError
+from ghapi.all import GhApi, paged
+
+
+class Check:
+    def __init__(self, api, org, repo):
+        raise NotImplementedError
+
+    def check(self):
+        raise NotImplementedError
```

Comment:
Nit: maybe `run()` instead of `check()` (given the class is `Check`)?

Reply:
Also would pair better with `dry_run`, though if they share enough logic, you could make it a param to `check` instead.

Reply:
Oh wait, I have this mixed up. `dry_run()` is for the `fix()` counterpart, and `check()` has no side-effects?